DEVELOPER.md

```diff
@@ -0,0 +1,374 @@
+# Developer Guide for TestSys
+
+## What is TestSys
+
+TestSys is a framework that leverages Kubernetes (k8s) to create resources and run tests.
+To learn more about how it works, see the [design](design/DESIGN.md) document.
+
+### Navigating the `bottlerocket-test-system` repo
+
+#### [Makefile.toml](Makefile.toml)
+
+TestSys uses make to unify the logic needed to build images.
```

Comment:
nit:
```suggestion
TestSys uses a makefile to unify the logic needed to build images.
```

I would even suggest removing this line, since the subsequent lines makes this point clear already.
```suggestion
```